components/tabs/README.md

```diff
@@ -37,77 +44,217 @@ Tabs are used to present related information in mutually exclusive panels, allow
 
 ## Tabs [d2l-tabs]
 
-The `d2l-tabs` element is a web component for tabbed content. It provides the `d2l-tab-panel` component for the content, renders tabs responsively, and provides virtual scrolling for large tab lists.
+The `d2l-tabs` element is a web component for tabbed content. It provides the `d2l-tab` component for simple tab content, and the `d2l-tab-panel` component for the panel content. It also provides the `TabMixin` for custom tabs. It renders tabs responsively and provides virtual scrolling for large tab lists. The tabs and panels are paired using an `id` on the tab handle and corresponding `labelled-by` on the `d2l-tab-panel`.
```

Comment:
Seems like we're overloading the term content a bit. What do you think of...
```suggestion
The `d2l-tabs` element is a web component for tabbed content, providing layout and responsive scrolling behaviour. The `d2l-tab` element is a simple tab handle that renders text, matched with a `d2l-tab-panel` element to contain the corresponding content. In addition, the `TabMixin` can be extended to implement custom tab handles. The tabs and panels are paired using an `id` on the tab handle and corresponding `labelled-by` on the `d2l-tab-panel`.
```

Please don't feel obligated to accept that ☝️  suggestion. I'm also sorta on the fence about mentioning `TabMixin`. Like menu-items, I want them to know it's possible, but I also kinda want them to reach out to before doing-so.

Reply:
There is also a section on "Custom Tabs" that documents the usage. I could see why we'd want to know the use case before it's used though. I'll think a bit more on how to document it in a subtle way.